alpha/books/RayTracingInOneWeekend.html

```diff
@@ -741,20 +745,22 @@
 Adding a Sphere
 ====================================================================================================
 Let’s add a single object to our ray tracer. People often use spheres in ray tracers because
-calculating whether a ray hits a sphere is pretty straightforward.
+calculating whether a ray hits a sphere is relatively simple.
 
 
 Ray-Sphere Intersection
 ------------------------
-Recall that the equation for a sphere of radius $r$ that is centered at the origin is:
+The equation for a sphere of radius $r$ that is centered at the origin is well known to be:
```

Comment:
Seems a bit condescending.

Reply:
Hmm. I got someone saying that they want us to explain where we got the equation of the sphere from.

I didn't want to do that. Let me figure out another way to say this.